compiler/ifcc.g4

```diff
@@ -2,40 +2,46 @@ grammar ifcc;
 
 axiom: prog;
 
-prog: fn+;
-fn: TYPE VARNAME '(' argsDef? ')' '{' content '}';
-content: (init | affectation | ifElse | whileDo | returnValue) content?;
+prog: fn+ EOF;
+fn: TYPE VARNAME '(' argsDef? ')' '{' content? '}';
+content: (init | affectation ';' | ifElse | whileDo | returnValue ';' | fnCall ';') content?;
 value: CONST | VARNAME | CHAR;
-returnValue: 'return' expression ';';
+returnValue: 'return' expression;
 init: TYPE declaration; 
 declaration: dec (',' dec)* ';' ; 
 dec: VARNAME ('=' expression)? ;
-affectation: VARNAME '=' expression ';' # affectationExpr;
+affectation: VARNAME '=' expression # affectationExpr;
 expression:
-	expression MULTDIV expression # expressionMultDiv
+	expression MULTDIVMOD expression # expressionMultDivMod
 	| expression ADDSUB expression # expressionAddSub
-	| expression '&=' expression # expressionAnd
-	| expression '|=' expression # expressionOr
+	| expression ('&=' | '&&') expression # expressionAnd
+	| expression ('|=' | '||') expression # expressionOr
 	| expression '^=' expression # expressionXor
 	| expression '==' expression # expressionEqual
 	| expression '!=' expression # expressionNotEqual
 	| expression '>' expression # expressionGreater
 	| expression '<' expression # expressionLess
+	| expression '>=' expression # expressionGreaterEqual
+	| expression '<=' expression # expressionLessEqual
 	| '(' expression ')' # expressionPar
-	| VARNAME '(' args? ')' # expressionFn
+	| fnCall # expressionFn
 	| value # expressionValue;
-ifElse: 'if' '(' expression ')' '{' content '}' ( 'else' '{' content '}' )?;
+fnCall: VARNAME '(' args? ')';
+ifElse: 'if' '(' expression ')' '{' content? '}' ( 'else' '{' content? '}' )?;
 whileDo: 'while' '(' expression ')' '{' content '}';
 args: (expression) (',' expression)*;
 argsDef: (TYPE VARNAME) (',' TYPE VARNAME)*;
 
-CONST: [0-9]+;
-ADDSUB: '+' | '-';
-MULTDIV: '*' | '/';
-CHAR: '\'' .? '\'';
 COMMENT: '/*' .*? '*/' -> skip;
+ONELINECOMMENT: '//' .*? '\n' -> skip;
```

Comment:
Idk if that's normal but you are not using this in the grammar

Reply:
I'm using it, to skip the one-line comments